woocommerce/woocommerce-ios
# [Elevate] Update linked products banner

Part of #7374, #7377.

## Description

This PR updates promo banner about cross-sells and upsells in product details view. It fixes all previous UI glitches by replacing SwiftUI implementation with UIKit cell, reusing the same viewmodel.
Appear/disappear actions are still WIP.

## Testing

1. Build and run the app in debug mode (to ensure the feature flag is enabled).
2. Open product *without* any linked products.
3. Confirm promo banner appears.
4. Tap "Try it now" and confirm it navigates to linked products screen.
5. Add some products, tap done to return - confirm banner disappears.

## Screenshot

before (SwiftUI) | after (UIKit)
--|--
![Simulator Screen Shot - 1](https://user-images.githubusercontent.com/3132438/183484029-93093c5a-c18c-4d53-ba40-94bb86b7b795.png)|![Simulator Screen Shot - 2](https://user-images.githubusercontent.com/3132438/183484041-b09e6eb6-cd18-47d8-9bb6-71fd777a8298.png)

---
